navapbc/wic-participant-recertification-portal
# PRP-225 - Routing management framework, session reset

## Ticket

https://wicmtdp.atlassian.net/browse/PRP-225

## Changes
* Completed submissions route to the /confirm page
* Clicking "start over" on the /confirm page will route back to the start and reset session
* Attempting to access pages out of order will route the user back to the correct page, without displaying an error


## Context for reviewers
* To test routing:
    *  Route to a later page from the index, like `/gallatin/recertify/contact`
    *  Get routed back to `/gallatin/recertify`
* To test completion:
    * Finish a submission, view the /confirm page
    * Hit the back button
    * See the warning about having already submitted the form, with the "start over" link
    * Click the "start over" link
    * Be routed back to the top of the form
    * See that you are able to fill out the form again
